### PR TITLE
Wood Variants

### LINKED ISSUE #148: Add Wood and Stripped Logs & Wood
**Describe the solution you'd like**
- Full Bark Logs, Stripped Logs, Stripped Wood, and other wood variants are currently missing from the mod.

### PR DESCRIPTION
**Pull request by WenXin20**
- Adds Woods, Stripped Logs, & Stripped Woods
- Adds stairs, slabs, fences/gates, pressure plates, buttons, doors, & trapdoors for all 9 planks.
- Berry logs can be stripped
___
- Fixed the loot table of the Aged Log
- Closes #148 
___
- ~Aged, Temporal, & Inverted logs cannot be stripped yet, need some help for that.~

### CODE REVIEW

File: src/main/java/pokecube/legends/init/BlockInit.java [hunk 1]
```diff
@@ -328,11 +347,33 @@ public static void init()
...
+        @SuppressWarnings("deprecation")
+		public PokecubeWoodStairs() {
+            super(Blocks.OAK_PLANKS.getDefaultState(), Block.Properties.from(Blocks.OAK_PLANKS)
+                    .sound(SoundType.WOOD)
+                    .hardnessAndResistance(2.0f, 3.0f)
+            );
+        }
+    }
+    
+//	public static void addStrippable(final RegistryObject<Block> log, RegistryObject<Block> strippedLog) {
```
Comment (WenXin20) [inline src/main/java/pokecube/legends/init/BlockInit.java]: This is the code block that isn't functional for stripping logs. The `.put` doesn't take the `RegistryObject<Block>`, any suggestions to fix this?

File: src/main/java/pokecube/legends/init/BlockInit.java [hunk 2]
```diff
@@ -328,11 +347,33 @@ public static void init()
...
+//		AxeItem.BLOCK_STRIPPING_MAP = Maps.newHashMap(AxeItem.BLOCK_STRIPPING_MAP);
+//		AxeItem.BLOCK_STRIPPING_MAP.put(log, strippedLog);
+//	}
+//	
+//	public void strippableBlocks(FMLCommonSetupEvent e) 
+//	{
+//		addStrippable(ULTRA_LOGUB01, STRIPPED_INVERTED_LOG);
+//		addStrippable(INVERTED_WOOD, STRIPPED_INVERTED_WOOD);
+//	}
+
```
Comment (WenXin20) [inline src/main/java/pokecube/legends/init/BlockInit.java]: This is the code block that isn't functional for stripping logs. The `.put` doesn't take the `RegistryObject<Block>`, any suggestions to fix this?

Comment (Thutmose) [reply src/main/java/pokecube/legends/init/BlockInit.java]: what does the `.put` take? you can get the block from the `RegistryObject<Block>` via `.get()`

**Comment by Thutmose** [comment]
Can you also do one for 1.16.4, or do the merge pull request for that side? 1.15.2 is currently on "bug fix and maintenance only" as I didn't want to have to deal with excessive merging back/forth between 1.16.4 and 1.15.2, since 1.16.4 has recently had some cleanup run.

I am willing to merge this into 1.15.2, and release there, but I don't really want to have to deal with trying to merge it up to the 1.16.4 branch

File: src/main/java/pokecube/legends/init/BlockInit.java (hunk 2 above)
Comment (WenXin20) [reply src/main/java/pokecube/legends/init/BlockInit.java]: Thanks! That fixed it.

**Comment by WenXin20** [comment]
Ok, I'll port it. Would that need a separate pr?

**Comment by Thutmose** [comment]
Ya, I think it would require a separate one, or a cherry-picked merge request, a separate one might be easier though.

**Comment by Thutmose** [comment]
So is this good to merge in to 1.15.2, and then you will do a PR for 1.16.4? if so, I will merge it in when I  see confirmation.

**Comment by WenXin20** [comment]
Yep, this one is done, and a separate PR will come for 1.16.4 later, maybe tonight or tomorrow.

**Pull request merged**